CHANGELOG

```diff
@@ -1,3 +1,11 @@
+Version 0.14.2 (2022-07-08)
+---------------------------
+Bump collector to 2.6.2 (#360)
+Bump Elasticsearch Loader to 1.0.7 (#361)
+Bump Enrich to 3.2.5 (#362)
+Fix links on overview page (#359)
+Update how gpg key is loaded when installing Caddy (#364)
+
```

Comment:
just a nit: this update shouldn't live in enrich version bump but in release commit

Reply:
Thanks for spotting this! I'll fix it now.